integration_tests/test_dataimport.py

```diff
@@ -67,10 +69,11 @@ def validate_mipmap_generated(in_ts,out_ts,levels,imgformat='tif'):
 
     for lvl, mmL in iteritems(out_ts.ip):
         fn = urllib.parse.unquote(urllib.parse.urlparse(mmL.imageUrl).path)
-        ext=os.path.splitext(fn)[1]
+        ext = os.path.splitext(fn)[1]
         if (lvl != "0"):
             assert ext[1:] == imgformat
         with Image.open(fn) as im:
+            assert im.mode == targetmode
```

Comment:
if we have this assert, why are we giving the option to set the targetmode with the default = 'L', shouldn't the default be to keep the mode of the input file, and if the targetmode is set, to set it to be the output mode.  If I recall correctly, render can handle correctly 16bit tiff mipmap 0 levels and 8bit jpeg mipmaps.

Reply:
This is a good point -- I am not sure from where the code coverage on converting from 16 bit to 8 bit is coming, but this change doesn't seem to affect it.  I'm not too concerned, but that's something that should probably be fixed..